redi-connect-front/src/pages/app/profile/Profile.tsx

```diff
@@ -45,14 +49,45 @@ function Profile ({ loading, profile, currentUser, profilesFetchOneStart }: Prof
     <>
```

Comment:
There's a simple improvement we can do here to improve readability a lot. Let's do this:
```
if (loading) return (<FullScreenCircle loading={loading} />)

return (
  <LoggedIn>
    blabla
  </LoggedIn>
)
```